L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -655,6 +657,22 @@ void l1t::GlobalBoard::initTriggerConditions(const edm::EventSetup& evSetup,
           }
         } break;
 
+        case CondTOPO: {
+          theCondition = std::make_unique<TOPOCondition>(itCond.second, this);
+          theCondition->setVerbosity(m_verbosity);
+
+          if (m_saveAXOScore and not m_axoScoreConditionName.empty()) {
```

Comment:
Is it right that this is using `m_saveAXOScore and not m_axoScoreConditionName.empty()`? should these be `m_saveTOPOScore and not m_topoScoreConditionName.empty()`

Reply:
indeed, though we were not sure we can actually store multiple different scores for different models. 
would this be possible?